core/src/main/java/io/fabric8/maven/core/handler/ProbeHandler.java

```diff
@@ -83,6 +83,8 @@ private HTTPGetAction getHTTPGetAction(String getUrl) {
     }
 
     private TCPSocketAction getTCPSocketAction(String port) {
+        // TODO what to do about host?
```

Comment:
How about this?
![probhandeler](https://user-images.githubusercontent.com/28615674/31424183-85b7f280-ae76-11e7-8e1b-89279582bd0e.png)

Reply:
sounds good to me. Fancy udpating this PR? :)